packages/sdk-core/src/data.ts

```diff
@@ -116,6 +121,109 @@ export class Datachannel extends EventEmitter {
     }
   }
 
+  public async requestStartPublishChannel(req: {
```

Comment:
We should leave all of details to msg_channel. I think in data.ts we only implement abstract like: requestMsgChannel

Reply:
Isn't `data.ts` purpose is to communicate to the server through implemented apis? `msg_channel` feature is depended on `data.ts`, so shouldn't the abstraction be on the `msg_channel` side? Developer should be able call `data.ts` in anyway they prefered without any abstraction.

Reply:
@luongngocminh No, data.ts is an abstraction for contacting servers. We don't need to update data.ts each time the protocol of Sender, Receiver, or MessageChannel ... changes.